composable_kernel/include/tensor_operation/element_wise_operation.hpp

```diff
@@ -141,6 +141,93 @@ struct AddHardswishAdd
     }
 };
 
+// Unary operators are usually called element-wisely before the reduction is executed on the
+// elements.
+// They are needed for easy implementation of reduction types of AVG, NRM1, NRM2
+
+template <typename Y, typename X, bool HasDividing = false>
+struct UnaryIdentic
+{
+    __host__ __device__ UnaryIdentic(const index_t divider = 1) { (void)divider; };
```

Comment:
This divider is used in computation with tensor data.

So its type should be ```int32_t```

https://github.com/ROCmSoftwarePlatform/composable_kernel/wiki/Coding-Style